src/export.rs

```diff
@@ -47,6 +47,25 @@ pub fn acquire() {
     unsafe { _defmt_acquire() }
 }
 
+#[cfg(feature = "unstable-test")]
+pub fn flush() {}
+
+#[cfg(not(feature = "unstable-test"))]
+#[inline(never)]
+pub fn flush() {
+    extern "Rust" {
+        fn _defmt_acquire();
+        fn _defmt_flush();
+        fn _defmt_release();
+    }
+    // SAFETY: ...
+    unsafe {
+        _defmt_acquire();
+        _defmt_flush();
+        _defmt_release()
+    }
+}
+
```

Comment:
Are these used by macros, or why are they in the `export` module?

Reply:
Do you only mean `flush` or all of `aquire`, `write`, `flush` and `release`?
And no, flush isn't used by default in some macro. I just placed them there to be consistent with the other logger-methods.
Do you know a better place?

Reply:
`export` is not part of our public API, it only contains methods that have to be `pub` so macros can refer to them. If users are supposed to call `defmt::flush()`, then it needs to be placed in `src/lib.rs`.

Reply:
I see. Was wondering about the purpose of export module anyways. Thanks for clarifying 🙈

Reply:
This was not yet resolved

Reply:
Is it now?